py/desisim/test/test_quickgen.py

```diff
@@ -122,11 +123,12 @@ def tearDownClass(cls):
             else:
                 os.environ[e] = cls.origEnv[e]
         if os.path.exists(cls.testDir):
+            pass
```

Comment:
Why is this here?

Reply:
Great catch, this was left over from testing. Done in https://github.com/desihub/desisim/pull/559/commits/a52762599ce169b2bec2d29ca29d2be3492fe53c.